pinc/3rdparty/iconify/octicon/README.md

```diff
@@ -0,0 +1,3 @@
+# Iconify
```

Comment:
I realize that Iconify has icons with different licenses, but I don't think we need to have a subdir for each class. Consider moving this contents up a level, renaming `LICENSE` `MIT-LICENSE` and the icons `octicon-<name>`.

Reply:
I'll do that.